metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1445,6 +1445,51 @@ def run_assertions_for_asset(
 
         return res["runAssertionsForAsset"]
 
+    def get_entities_v2(
```

Comment:
any chance we should brand this more specifically toward open api? (more a question for harshal)